ui/webpack.config.js

```diff
@@ -1,16 +1,16 @@
 const path = require('path');
 
-const groupKind = 'argoproj.io/Rollout';
+const extName = "Rollout";
 
 const config = {
   entry: {
     extension: './src/index.tsx',
   },
   output: {
-    filename: 'extensions.js',
-    path: __dirname + `/dist/resources/${groupKind}/ui`,
-    libraryTarget: 'window',
-    library: ['extensions', 'resources', groupKind],
+    filename: `extensions-${extName}.js`,
+    path: __dirname + `/dist/resources/extension-${extName}.js`,
+    libraryTarget: "window",
+    library: ["tmp", "extensions"],
```

Comment:
Not sure what is usage here? Can you briefly explain

Reply:
I add the same id of Metrics Extension https://github.com/argoproj-labs/argocd-extension-metrics/blob/5d714b0b79453a3eb6c0315c8294e0a3fcb4fbdb/extensions/resource-metrics/resource-metrics-extention/ui/webpack.config.js#L16